jetstream/outcomes/firefox_desktop/networking_performance.toml

```
@@ -0,0 +1,86 @@
+friendly_name = "Networking Performance"
+description = "Network layer performance metrics"
```

I'd suggest something that indicates that this is a subset of the other outcome.
```suggestion
description = "Network layer performance metrics (subset of Networking outcome without JS performance metrics)"
```

---

Perfect, I've updated it with a comment regarding the subseting.